hellomama_registration/utils.py

```diff
@@ -89,24 +105,17 @@ def get_identity(identity):
 
 
 def get_identity_address(identity):
-    url = "%s/%s/%s/addresses/msisdn" % (settings.IDENTITY_STORE_URL,
-                                         "identities", identity)
-    params = {"default": True}
-    headers = {
-        'Authorization': 'Token %s' % settings.IDENTITY_STORE_TOKEN,
-        'Content-Type': 'application/json'
-    }
-    r = session.get(url, params=params, headers=headers).json()
-    if len(r["results"]) > 0:
-        return r["results"][0]["address"]
-    else:
-        return None
+    return identity_store_client.get_identity_address(identity)
 
 
-def search_identities(params):
```

Comment:
Why the change the to the arguments here?

Reply:
Ah sorry I put it in the commit message but didn't copy it into the pull request:

This behaviour more closely matches the seed-services API client so it will make it easier to use the API client when that supports pagination.